File: python/ch01/02.chat.py
from langchain_ollama import ChatOllama
from langchain_core.messages import HumanMessage

model = ChatOllama(model='llama3.2')
prompt = [HumanMessage('프랑스의 수도는 어디인가요?')]

response = model.invoke(prompt)
print(response.content)

File: javascript/ch01/02.chat.js
import { ChatOpenAI } from '@langchain/openai';
import { HumanMessage } from '@langchain/core/messages';

const model = new ChatOpenAI();
const prompt = [new HumanMessage('프랑스의 수도는 어디인가요?')];

const response = await model.invoke(prompt);
console.log(response);
